### PR TITLE
fix MANAGER-9061 (pci.notebooks): display stop modal when stopping a notebook from list too

### PR DESCRIPTION
**Pull request by jperchoc**
~~### Do not merge into `fix/MLS-2703-notebook-informations-resources`~~
~~Instead:~~
~~- merge `fix/MLS-2703-notebook-informations-resources`  into master first~~
~~- then merge this PR into master~~

MANAGER-9061

Signed-off-by: Jonathan Perchoc <jonathan.perchoc@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `fix/MLS-2703-notebook-informations-resources` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-9061
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Move stop notebook modal to a component, to be able to use it in both the notebooks list and a notebook's dashboard.

### :flags: Translations

- [x] TRANS-43374

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/notebooks/stop-notebook/stop-notebook.module.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
```
Comment (Dimfacion) [inline packages/manager/modules/pci/src/projects/project/notebooks/stop-notebook/stop-notebook.module.js:5]: Are you sure all these imports are needed ?

Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/notebooks/stop-notebook/stop-notebook.module.js]: Looking at the `delete-notebook.module.js` file, it seems that we don't have to import `@ovh-ux/ng-translate-async-loader`,  `angular-translate` and `@ovh-ux/ui-kit`. 
I've removed them those imports here: https://github.com/ovh/manager/pull/6831/commits/86e0de1243f75dd4d3a0a96671f3d2af50b57c3e

Comment (Dimfacion) [reply packages/manager/modules/pci/src/projects/project/notebooks/stop-notebook/stop-notebook.module.js]: lgtm then :)

**Review by Dimfacion — DISMISSED** [review]

**Review by Dimfacion — APPROVED** [review]

**Review by marie-j — APPROVED** [review]

**Comment by marie-j** [comment]
Rebased on develop

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**